index.html

```diff
@@ -0,0 +1,140 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>Katherine Kocen | Protfolio</title>
+<!-- Links to CSS Style Sheets and Fonts -->
+            <link rel="stylesheet" href="css/main.css">
+            <link rel="stylesheet" href="css/normalize.css">
+            <link href="https://fonts.googleapis.com/css2?family=Lora:wght@400;700&display=swap" rel="stylesheet">
+            <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap" rel="stylesheet">
+    </head>
+
+
+    <body>
+        <header>
+            <div class="top">
+                <div class="full-width">
+                    <nav>
+                        <ul>
+                            <li><a href="#home">Home</a></li>
+                            <li><a href="#portfolio">Portfolio</a></li>
+                            <li><a href="#about">About</a></li>
+                            <li><a href="#contact">Contact</a></li>
+                        </ul>
+                    </nav>
+                </div>    
+            </div>
+
+            <div class="cover">
+                <div class="full-width">
+                    <h1>Hi There! I'm Katherine Kocen.</h1>
```

Comment:
_nit_ I think having `There` be capitalized too is a little strange given this section has sentence casing bc of its punctuation 🙈 
https://blog.prototypr.io/sentence-case-or-title-case-5bd581f05d88?gi=43d26eb86d0